cdap-docs/_common/common-build.sh

```diff
@@ -358,7 +367,57 @@ function display_version() {
   echo "PROJECT_SHORT_VERSION: ${PROJECT_SHORT_VERSION}"
   echo "GIT_BRANCH: ${GIT_BRANCH}"
   echo "GIT_BRANCH_TYPE: ${GIT_BRANCH_TYPE}"
-  echo ""
+}
+
+function set_message_file() {
+  TMP_MESSAGE_FILE="/tmp/$(basename $0).$$.tmp"
+  export TMP_MESSAGE_FILE
+}
```

Comment:
If you're going to do this, you'll want some form of `cleanup_tmpfiles` function, which gets called unconditionally, before exit, after messages are replayed.